pkg/gofr/http/response/response.go

```diff
@@ -0,0 +1,21 @@
+package response
+
+import (
+	"net/http"
+)
+
+type Response struct {
+	Data    any               `json:"data"`
+	Headers map[string]string `json:"-"`
```

Comment:
Maybe rename to this

```suggestion
	CustomHeaders map[string]string `json:"-"`
```

This way the variable and method SetCustomHeaders names will be consistent.

Just a suggestion, it's up to you